css-display-4/Overview.bs

```diff
@@ -1163,6 +1198,39 @@ Reading Order: the 'reading-order' property</h3>
 
 	ISSUE: Is this the most appropriate interaction with with <code>tabindex</code>?
 
+<h4 id="randomizing-layout">
+Randomizing Layout Methods and ''reading-order: auto''</h4>
+
+	<dfn>Randomizing formatting contexts</dfn>
+	are formatting contexts that can indiscriminately scramble the layout order,
+	as listed below.
+	Since use of such methods indicates that the underlying order of the elements
+	is not important,
+	the UA will match the [=reading and navigation order=] of the elements
+	to match the layout order of their boxes as follows:
+
+	<dl>
```

Comment:
What else will be included here? Masonry for grid seems likely. What about flex-direction: *-reverse?

Reply:
Only randomized orders, so masonry might (does it actually randomize?) but flex-direction: *-reverse wouldn't.